.github/workflows/fullTest.yml

```diff
@@ -8,15 +8,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ 11, 15 ]
+        java: [ 11, 15, 17 ]
```

Comment:
i think we can remove 15 and only keep the lts versions...
but add 18 as additional in time...

Reply:
11 and 17 makes sense as both are LTS releases